From c656721bd97fdffb9bce434817ee009b6ef3f9aa Mon Sep 17 00:00:00 2001
From: Daniel Scheller <d.scheller@gmx.net>
Date: Sun, 15 Oct 2017 18:29:33 +0200
Subject: [PATCH 4/8] [media] ddbridge/ci: change debug printing to debug
 severity

slot_ts_enable_xo2() logged debug output to info instead of debug, so
fix this up.

Signed-off-by: Daniel Scheller <d.scheller@gmx.net>
---
 drivers/media/pci/ddbridge/ddbridge-ci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/pci/ddbridge/ddbridge-ci.c b/drivers/media/pci/ddbridge/ddbridge-ci.c
index c775b17c3228..457c711aaced 100644
--- a/drivers/media/pci/ddbridge/ddbridge-ci.c
+++ b/drivers/media/pci/ddbridge/ddbridge-ci.c
@@ -260,7 +260,7 @@ static int slot_ts_enable_xo2(struct dvb_ca_en50221 *ca, int slot)
 {
 	struct ddb_ci *ci = ca->data;
 
-	dev_info(ci->port->dev->dev, "%s\n", __func__);
+	dev_dbg(ci->port->dev->dev, "%s\n", __func__);
 	write_creg(ci, 0x00, 0x10);
 	return 0;
 }
-- 
2.13.6

